src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -175,13 +185,17 @@ public void periodic() {
                 Rotation2d.fromDegrees(currentPose.getHeadingDegrees()));
         Logger.getInstance().recordOutput("odometry", currentPose2d);
 
-        // Push robot info to Dashboard
-        SmartDashboard.putNumber("Kinematics X (Feet)", currentPose.getX());
-        SmartDashboard.putNumber("Kinematics Y (Feet)", currentPose.getY());
-        SmartDashboard.putNumber("Kinematics Heading (degrees)", currentPose.getHeadingDegrees());
+        // // Push robot info to Dashboard
+        // SmartDashboard.putNumber("Kinematics X (Feet)", currentPose.getX());
```

Comment:
If we aren't using this remove the commented out code

Reply:
Removed